### ISSUE TITLE
Missing inline block comments

### ISSUE THREAD

**Issue by Righolt** [VaccineAndDrugEvaluationCentre/code-diary-sas#8]
Not all block comment options are currently parsed right.

Code Diary is not parsing
```
/**Same line Code Diary block comment A*/
```
or
```
/**
Same line Code Diary block comment C (not implemented)*/
```

Both are included in the demo for testing.

**Comment by Righolt** [comment]
@rbisewski  If I remember correctly the start and end of line regular expressions in SAS had issues. Did you encounter that?

In particulare code_diary.sas lines 313-315
```
* if starts with "slash-two-asterix-then-at" AND ends with "asterix-slash";
* then convert it to a normal starting "two-star" line;
source_line = prxchange('s/^.*;?\s{0,4}\/\*\*\@(.+)\*\//\*\*\@$1;/', -1, source_line);
```
Do not seem to work as intended (comment A above). Have `^` and `$` to indicate the start and end of line worked for you in SAS?

**Comment by rbisewski** [comment]
As per above, I have created a branch with the patches necessary to make this work.

Let me know what you think.

**Comment by Righolt** [comment]
Solved #10
